server/utils.go

```diff
@@ -397,6 +399,41 @@ func versionToString(vers int) string {
 	return str
 }
 
+// rewriteToken attempts to match the original token
+// against the email, telephone number or login patterns.
+// On success, prepends the token with the corresponding prefix.
+func rewriteToken(orig string) string {
+	if orig == "" || strings.HasPrefix(orig, "basic:") ||
+			strings.HasPrefix(orig, "email:") || strings.HasPrefix(orig, "tel:") {
```

Comment:
I would just test for any prefix at all. I.e. if ```alphanum:``` is present then don't rewrite. If the user knows about prefixes then we should not interfere.

Reply:
Done.